Say Their Names/Source/Controller/Home/HomeView.swift

```diff
@@ -37,9 +37,16 @@ final class HomeView: UIView {
         return peopleCollectionView
     }()
     
+    let bookmarkButton: UIButton = {
```

Comment:
Maybe we should have our own button now: `UIImageButton` or something?

Reply:
Great suggestion. Going to add this as a separate task as I don't see it as being a blocker for this commit and would be a great task for a first timer